### PR TITLE
ISO-object used for integration testing in ATOS

### PR DESCRIPTION
**Pull request by samuelthoren**
I have made an ISO-object that when connected gets the trajectory to use, sets it location to the first point of the trajectory when armed, and then sends the trajectory points in a MONR-message when in the running state. This makes it so that we can run integration tests automatically.

### CODE REVIEW

**Review by sepast — APPROVED** [review]
Looks good
Nice to have while testing

**Comment by Robert108** [comment]
Not so sure that we should add ATOS specific functionality to this repo. I'd rather see that we keep this as a clean reference implementation. Maybe a fork would fit better?

**Comment by Robert108** [comment]
> > Not so sure that we should add ATOS specific functionality to this repo. I'd rather see that we keep this as a clean reference implementation. Maybe a fork would fit better?
> 
> Yes you are probably right, but I think it would be harder to find this functionality if we make it a fork. Maybe we can add an example directory with some examples on how to use the ISO-object, and I can put my code there instead?

I took another look and I think the feature you implemented here could very well be the default behavior for the IsoObject demo executable instead of the circling around the origin. But if we or an other user want to change the behavior we could add a flag instead of needing to build a seperate execuable.

**Comment by samuelthoren** [comment]
Implemented the things we discussed on Teams @Robert108. Also fixed the heading when running in a circle.

File: demoIsoObject.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@ static po::variables_map parseArguments(
 	desc.add_options()
 		("help,h", "print this message")
 		("listen-ip,i", po::value<std::string>()->default_value("0.0.0.0"), "The IP address that the isoObject will listen on.")
+        ("behaviour,b", po::value<std::string>()->default_value("default"), "The behaviour of the isoObject. Options are 'default' and 'circle'")
```
Comment (victorjarlow) [inline demoIsoObject.cpp:26]: Maybe this can be called "follow-trajectory" or similar? instead of default, might be more descriptive.

**Review by victorjarlow — APPROVED** [review]
minor comments

**Pull request merged**